frontend/freebase2wikidata.js

```diff
@@ -2018,7 +2018,8 @@ $(function() {
         return reportError(error);
       }).done(function() {
         if (this.statement.source.length > 0) {
-          return; // TODO add support of source review
+          // TODO add support of source review
+          widget.toggle(false).setDisabled(true);
         }
```

Comment:
You should keep the return here in order to don't tag the source as approved in the primary sources database